src/components/adaptive-cards/AdaptiveCard/AdaptiveCard.stories.js

```diff
@@ -1860,6 +1860,39 @@ const exampleActionOpenUrl = {
   ],
 };
 
+const exampleActionShowCard = {
+  $schema: 'http://adaptivecards.io/schemas/adaptive-card.json',
+  type: 'AdaptiveCard',
+  version: '1.0',
+  body: [
+    {
+      type: 'TextBlock',
+      text: 'This card\'s action will show another card',
+    },
+  ],
+  actions: [
```

Comment:
let's have two cards here, to demonstrate the multiple card functionality: "Show card 1", "Show card 2"